src/css/components/card.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../../node_modules/uswds/src/stylesheets/core/defaults.scss';
+@import '../override_vars.scss';
+@import '../core.scss';
+
+.card-home-contents {
```

Comment:
FYI, I don't agree that we should rename the feature 'card' — the feature is still this specific implementation/variant of the card.